charts/trino/templates/deployment-coordinator.yaml

```diff
@@ -20,6 +20,8 @@ spec:
         app: {{ template "trino.name" . }}
         release: {{ .Release.Name }}
         component: coordinator
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap-coordinator.yaml") . | sha256sum }}
```

Comment:
We need to trigger new deployments on ANY change, not just these two config maps.

I think existing behaviour where operator decides to do fresh deployment should be kept as it is for now.

We can add feature in separate PR to trigger deployment when any catalogs change or node configs change.